packages/frontend/src/app/app-component/config/config.component.ts

```diff
@@ -12,30 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Component} from '@angular/core';
+import {Component, Input} from '@angular/core';
 import {UserService} from 'src/app/services/user/user.service';
+import {environment} from 'src/environments/environment';
 
 @Component({
   selector: 'app-config',
   templateUrl: './config.component.html',
   styleUrls: ['./config.component.css'],
 })
 export class ConfigComponent {
-  linkedGithubAccounts = [
-    {name: 'account 1'},
-    {name: 'account 2'},
-    {name: 'account 3'},
-    {name: 'account 4'},
-    {name: 'account 5'},
-  ];
+  @Input() repoName: string;
+  @Input() orgName: string;
 
-  token = 'MF93N60H857HBGN65GHBN035GHB0PJN605';
-  showToken = false;
-  showAdminView = false;
+  exportUrl = '';
+  windowProvider = window;
 
   constructor(public userService: UserService) {}
 
-  onTokenClick() {
-    this.showToken = !this.showToken;
+  onDownloadClick() {
+    this.exportUrl =
```

Comment:
Since this is used only here, maybe we can create it here instead of having it as an attribute of the class ?
Like:
`const exportUrl = ...`

Reply:
That is a good idea! I will do that instead, thank you